quinn-udp/src/unix.rs

```diff
@@ -230,8 +262,18 @@ fn send(
     let num_transmits = transmits.len().min(BATCH_SIZE);
 
     loop {
+        #[cfg(not(target_os = "freebsd"))]
         let n =
             unsafe { libc::sendmmsg(io.as_raw_fd(), msgs.as_mut_ptr(), num_transmits as u32, 0) };
+        #[cfg(target_os = "freebsd")]
+        let n = unsafe {
+            libc::sendmmsg(
+                io.as_raw_fd(),
+                msgs.as_mut_ptr(),
+                num_transmits as libc::size_t,
+                0,
+            )
+        };
```

Comment:
Can we fold these together by using `as _`, or at least isolate the cfg to declaring an intermediate variable? It took me a solid thirty seconds to find the difference between these two lines.

Reply:
You are right, let's use ```as _```.

Reply:
Looks like this hasn't been done yet.

Reply:
Sorry, I reverted too much last time.